mydemoproject/demoapp1/management/commands/send_old_user_password_reset.py

```diff
@@ -0,0 +1,53 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.urls import reverse
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+from django.conf import settings
+from django.template.loader import render_to_string
+from django.core.mail import send_mail
+from django.utils.html import strip_tags
+from demoapp1.models.user import User
+
+
+class Command(BaseCommand):
+    help = 'Send password reset emails to users with the role of olduser.'
+
+    def handle(self, *args, **kwargs):
+        # Filter users with the role of 'olduser'
+        old_users = User.objects.filter(role='olduser')
```

Comment:
We should not identify using role because it should be `User`. Both old and new should be `User`.